nursery/mixed-mode-dotnet.yml

```diff
@@ -0,0 +1,9 @@
+rule:
+  meta:
+    name: mixed mode dotnet
```

Comment:
would be helpful to add a description of what this means

Reply:
added